### PR TITLE
Adding prow config for release-1.2 branch

### PR DESCRIPTION
**Pull request by icygalz**
Adding prow config for release-1.2 branch.

### CODE REVIEW

File: prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+    - name: istio-integ-k8s-tests
+      <<: *job_template
+      labels:
+        preset-service-account: "true"
+      spec:
+        containers:
+        - <<: *istio_container
+          command:
+          - entrypoint
+          - prow/istio-integ-k8s-tests.sh
```
Comment (howardjohn) [inline prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml:443]: This should be base on master not release-1.1 I think. We no longer have this target in master (which is now 1.2

Comment (icygalz) [reply prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml:443]: That target still exist in:
https://github.com/istio/test-infra/blob/master/prow/cluster/jobs/istio/istio/istio.istio.master.yaml#L443

File: prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+        command:
+        - entrypoint
+        - prow/istio-presubmit.sh
+      nodeSelector:
+        testing: build-pool
+    run_after_success:
+    - name: istio-integ-k8s-tests
+      <<: *job_template
+      optional: true
+      context: prow/istio-integ-k8s-tests.sh
```
Comment (howardjohn) [inline prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml]: Sorry I meant this one , presubmit. We split to things like prow/integ-galley-k8s-presubmit-tests.sh

Comment (icygalz) [reply prow/cluster/jobs/istio/istio/istio.istio.release-1.2.yaml]: updated.

**Pull request merged**